apps/OpenSpace/ext/launcher/include/sgctedit/settingswidget.h

```diff
@@ -72,17 +74,81 @@ Q_OBJECT
      */
     void setShowUiOnFirstWindow(bool setUiOnFirstWindow);
 
+    /**
+     * Sets value for whether or not the checkbox for having the UI only on the first
+     * window is enabled. This checkbox should only be clickable if the number of
+     * windows is 2 or more. 
+     */
+    void setEnableShowUiOnFirstWindowCheckbox(bool enable);
+
+    /**
+     * Gets the value of the selection for which display first window should mirror if
+     * the option to show the Ui in the first window is enabled. Note that this will
+     * return a value even if the checkbox is not enabled.
+     * 
+     * \return -1 if in a disabled state (e.g. when showUiOnFirstWindow() returns false)
+     *          0 if no window is being mirrored (only the UI will appear in first
+     *            window, with no 3D rendering)
+     *          1 if the second window's viewport is mirrored in the first window
+     *          2 if the third window's viewport is mirrored in the first window
+     *          3 if the fourth window's viewport is mirrored in the first window
+     */
+    int graphicsSelectionForShowUiOnFirstWindow() const;
+
+    /**
+     * Sets value of the graphics selection combo box for which other window that the
+     * first window will mirror.
+     * 
+     * \param selection int value for the combo box selection.
+     *                  0 if no window is being mirrored (only the UI will appear)
+     *                  1 if the second window's viewport is mirrored in the first window
+     *                  2 if the third window's viewport is mirrored in the first window
+     *                  3 if the fourth window's viewport is mirrored in the first window
+     */
+    void setGraphicsSelectionForShowUiOnFirstWindow(int selection);
```

Comment:
Same here

Reply:
Same response as for the getter function above